### PR TITLE
[Components with more than 1 service] LPS-178221 Refactor UpgradeStepRegistrator components with more than 1 service type

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
ci:test:upgrade

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

**Review by rafaprax — APPROVED** [review]

**Comment by tinatian** [comment]
@MrJohn1911 @rafaprax Please see my comment and resend.

Thanks.
Tina

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/portlet/CalendarPortlet.java [hunk 1]
```diff
@@ -1853,10 +1852,6 @@ private CalendarBooking _updateCalendarBooking(
 	@Reference
 	private CalendarService _calendarService;
 
-	@Reference
-	private CalendarWebUpgradeStepRegistrator
```
[the remark this answers is not in the document]
Comment (rafaprax) [reply modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/portlet/CalendarPortlet.java:1857]: Hey @tinatian, it already has a dependency on release, see https://github.com/MrJohn1911/liferay-portal/blob/1bf9cb2bd390b165df393e504ea32f5d33a9d60f/modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/portlet/CalendarPortlet.java#L1876-L1880

**Comment by tinatian** [comment]
@MrJohn1911 @rafaprax Please update the commit message to describe the reason it is safe to remove.

Thanks.
Tina.

**Comment by MrJohn1911** [comment]
Hi @tinatian, in this last push, I just updated the commit message.

CC: @rafaprax

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/133036

**Pull request closed without merging**